redaxo/src/addons/metainfo/lib/input.php

```diff
@@ -46,6 +48,8 @@ public function setAttribute($name, $value)
 
     /**
      * Gibt den Wert des Attributes $name zurueck falls vorhanden, sonst $default.
+     *
+     * @return string|null
```

Comment:
Man könnte auch überlegen, mit echten return types zu arbeiten, da es eine interne Klasse ist.
Erstmal nur als phpdoc, da unkritischer, ist für mich aber auch ok! Vielleicht auch besser, gerade weil 5.8 schon in beta phase ist, und ich eher keine beta2 plane (wenn nicht noch größere Probleme auftauchen mit beta1).